projects/plugins/jetpack/class.jetpack-idc.php

```diff
@@ -1,722 +0,0 @@
-<?php
```

Comment:
I would recommend keeping the file around for a few releases, to avoid any Fatals during the plugin update for folks with aggressive cache setups.
See this example: 
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/class.jetpack-affiliate.php#L1-L5

Reply:
will do thanks for looking out for and the great feedback.

Reply:
Are we going to update the functions within to call the canonical version in this PR or a follow-up?

Reply:
@kraftbj this file shouldn't be loaded/used. As Jeremy outline it is only here to avoid fatals, I don't see a need to have these functions reference the package, but I may be overlooking use cases.

Reply:
Spent a little time thinking this through and as discussed in slack there are no known references to the Jetpack_IDC class. The intent here is to ensure no fatals during upgrade processes which is why the docblock is added. If another plugin references Jetpack_IDC directly it will fail, the only reason this would be available was if they include the file directly which would potentially cause a conflict if both the new and old IDC classes tried to add the same ui elements.

As such I'm going to delete the contents of the file in full.